EventFilter/GEMRawToDigi/python/GEMOrconCabling_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+from CondCore.DBCommon.CondDBSetup_cfi import *
+RPCCabling = cms.ESSource("PoolDBESSource",
+    CondDBSetup,
+    toGet = cms.VPSet(cms.PSet(
+        record = cms.string('RPCEMapRcd'),
+        tag = cms.string('RPCEMap_v2')
+    )),
+    connect = cms.string('oracle://orcon/CMS_COND_ON_200_RPC')
```

Comment:
What is  RPC doing here?

Reply:
thanks sven.
this isnt need and I will remove this file